web/src/user_group_popover.ts

```diff
@@ -53,17 +53,19 @@ export function handle_keyboard(key: string): void {
 }
 
 // element is the target element to pop off of;
-// message_id is the message id containing it, which should be selected;
+// the element could be user group pill or metions in a message;
+// in case of message, message_id is the message id containg it;
+// in case of user group pill, message_id is not used;
 export function toggle_user_group_info_popover(
     element: ReferenceElement,
-    message_id: number,
+    message_id: number | undefined,
```

Comment:
Original review:
> sahil: Do we need this change? We would anyway pass element and that can be used to get the group object and message_id can just be passed as undefined.

I agree, refactored the function to accept user group pill as element.